base_layer/core/src/transaction.rs

```diff
@@ -24,10 +24,24 @@
 // Version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0.
 
 use crate::{
+    block::AggregateBody,
     range_proof::RangeProof,
-    types::{Commitment, Signature},
+    types::{Base, BlindingFactor, Commitment, Signature},
 };
 
+use byteorder::{LittleEndian, WriteBytesExt};
```

Comment:
fyi, Edition 2018 has [endianness builtin](https://doc.rust-lang.org/std/primitive.u64.html#method.from_le_bytes). Is there any way to drop this dependency?

Reply:
Oh wow yes that is what that dependency does, must be new because searching for this functionality only returned material about the `byteorder`. It is also what Dalek uses.

Fixed.